src/pages/Younguk/List/CoffeeCard.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHeart as regularHeart } from "@fortawesome/free-regular-svg-icons";
+import { faHeart as solidHeart } from "@fortawesome/free-solid-svg-icons";
+import { Link } from 'react-router-dom';
+import { useState, } from 'react';
+
+function CoffeeCard(props) {
+
+    const [heartshape, heartshapeChange] = useState(regularHeart);
+    function click() {
+        if (heartshape === regularHeart) {
+            heartshapeChange(solidHeart);
+        }
+        else {
+            heartshapeChange(regularHeart);
+        }
+    }
+
+    return (
+        <li><Link to="/Detail"><img src={props.item.imgsrc} alt="커피이미지" /></Link>
```

Comment:
단계를 나눠주세요! 그리고 Link도 좋지만 useNavigate 방법을 쓰는 건 어떨까요?
```suggestion
        <li>
            <Link to="/Detail">
                <img src={props.item.imgsrc} alt="커피이미지" />
            </Link>
```

Reply:
useNavigate는 뭔가 정보가 왔다갔다 할때 쓰고, 단순히 웹페이지 이동이라면 Link가 더 좋을 것 같습니다.